### PR TITLE
[V9] Port of module V8

### PR DESCRIPTION
**Pull request by vrenaville**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Aren't you going to remove it?

**Comment by vrenaville** [comment]
@pedrobaeza sorry after discussion with @yvaucher it still make sense to have this module
(we need to have the buttons).

**Comment by pedrobaeza** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:  just the version to update

File: account_draft_invoice_print/__openerp__.py [hunk 1]
```diff
@@ -27,6 +27,6 @@
  'license': 'AGPL-3',
```
Comment (yvaucher) [inline account_draft_invoice_print/__openerp__.py:27]: Can you change the version to `9.0.1.0.0` ?

**Comment by vrenaville** [comment]
@yvaucher thanks for your comment, I have correct it according to your comment

**Comment by yvaucher** [comment]
@vrenaville Thanks

**Pull request merged**